services/eventService/eventService.go

```diff
@@ -27,11 +27,11 @@ type Event struct {
 }
 
 func (e *Event) GetInterestLine() string {
+	totalCount := e.AttendingCount + e.InterestedCount
 	if e.StartTime.After(time.Now()) || e.EndTime.After(time.Now()) {
-		totalCount := e.AttendingCount + e.InterestedCount
 		return fmt.Sprintf("%d going or interested", totalCount)
 	}
-	return fmt.Sprintf("%d attended", e.AttendingCount)
+	return fmt.Sprintf("%d went or interested", totalCount)
```

Comment:
Replace `"%d went or interested"` with `"%d went or were interested"`. You could also add (sad) special cases for `"1 went or was interested"` and `"No one went or was interested"`.

Reply:
:+1: on 'went or were interested'. I don't think (Sad) special cases are necessary, they will rarely, if ever, occur

Reply:
Since we're not completely sure that they won't occur, what happens when they do occur? Do we just ignore the permanent misspellings that will exist on the site? ;-)